pkg/controller/operators/catalog/operator.go

```diff
@@ -139,7 +142,7 @@ func (o *Operator) syncCatalogSources(obj interface{}) (syncError error) {
 	defer o.sourcesLock.Unlock()
 	o.sources[catsrc.GetName()] = src
 	o.sourcesLastUpdate = timeNow()
-	return err
+	return nil
```

Comment:
We mix this convention in the codebase (we know `err == nil`, but return `err` anyway). Which is preferred?

Reply:
I feel this is clearer, but I don't know if there's a golang convention for this. Which do you think is more readable?